### ISSUE TITLE
XPS 13 9370 Support

### ISSUE THREAD

**Issue by ikrananka** [kevinshroff/KSMRD-Modded-Realtek-Audio-Drivers#6]
I see that you show the XPS 13 9370 as "probably" supported.  Here's the hardware ID (second line) for the XPS 13 9370 that I have just received from Dell.  Does this help determine if your driver supports this hardware?

INTELAUDIO\FUNC_01&VEN_10EC&DEV_0299&SUBSYS_102807E6

**Comment by kevinshroff** [comment]
Yeah, just checked and that hardware ID is present. Should work, test and let me know

**Comment by moriel5** [comment]
I'm extremely sorry I had not posted the hardware ID on the Vostro 3568, I'll do so when I have the time.

And since I work as a computer technician, I'll post the audio hardware ID's of Dell PC's that go through our lab (provided that they are new enough that they use Realtek+Waves drivers).
